core/domain/suggestion_jobs_one_off.py

```diff
@@ -471,3 +475,74 @@ def reduce(key, values):
             yield (key, len(values))
         else:
             yield (key, values)
+
+
+class DeleteAllQuestionAndInvalidSuggestionsOneOffJob(
```

Comment:
Is this a job that we will only need once? If so, please specify in the documentation & explain why it's needed (a link to a findings document would be really helpful here for context).

Reply:
Yeah, this is only needed for once. I'll revert this PR once it's cherry-picked on the release branch and deployed. I've added a few notes in the docstring regarding why this deletion is needed.

Reply:
Can you specifically update the docstring to indicate that the job is only needed once and will soon be deleted? This just helps ensure it isn't misconstrued as a long-term job (especially in the event it does need to stay in the codebase for longer than we anticipate, e.g. due to other observed issues found when we continue the investigation/fix process).

Reply:
Done! (Also created #11300 for this task)

Reply:
I have updated the docstring.